src/ripple/app/ledger/Ledger.cpp

```diff
@@ -527,7 +527,7 @@ Ledger::rawInsert(std::shared_ptr<SLE> const& sle)
     auto item = std::make_shared<
         SHAMapItem const>(sle->key(),
             std::move(ss));
-    // VFALCO NOTE addGiveItem should take ownership
+    // addGiveItem should take ownership
```

Comment:
This comment makes no sense to me. It _should_ but _does it_? A better comment might be:

    // addGiveItem takes ownership

But maybe that isn't even needed, given the `std::move`. It may be clearer still to cut out the `std::move` and eliminate the `item` temporary entirely by inlining the call into `addGiveItem` which makes it extra obvious that the pointer is being handed off to the map for safe-keeping.

Reply:
Fixed in `bd4cc1711 [fold] Misc fixes:`